### ISSUE TITLE
Add changelog

### ISSUE THREAD

**Issue by demurgos** [tectonic-typesetting/tectonic#137]
Hi,
I am following this project and checking the repo from time to time. I see that there was a new release recently but finding what changed is not easy. I could inspect each commit or the closed issues / PRs but for a quick overview, it's a bit hard.

Would it be possible to add a `CHANGELOG.md` file to the repo to view the changes for each version?

**Comment by demurgos** [comment]
I just wanted to say that since my first post, I noticed that there are release posts on the forum: they list the main changes.

Since I check the repo more often than the forum, this is why I did not see it.

I still think that it would be nice to have a `CHANGELOG.md`: the forum post could just copy the main changes from the changelog.

**Comment by pkgw** [comment]
Yes, agreed. I was thinking that I might just have a CHANGELOG file that points to the forum posts.

**Comment by demurgos** [comment]
Thanks!
